static/js/hotkey.js

```diff
@@ -515,6 +515,12 @@ exports.process_hotkey = function (e, hotkey) {
         }
     }
 
+    // Prevents up and down navigation along the message list when lightbox is open.
+    if ((event_name === 'up_arrow' || event_name === 'down_arrow') && overlays.lightbox_open()) {
+        e.preventDefault();
```

Comment:
I don't think you need the `preventDefault()` here.

More importantly, this fix only addresses this specific issue, and we should be fixing the 15 similar issues that might exist.   

I think we actually want to put a `overlays.is_active` check just before the "Navigation shortcuts" section, that just returns false there regardless of the event name.